src/components/plots/plot.ts

```diff
@@ -54,6 +56,8 @@ module Plottable {
       this.clipPathEnabled = true;
       this.classed("plot", true);
       this._key2PlotDatasetKey = d3.map();
+      this._attrToExtents = d3.map();
+      this._extentProvider = (scale: Scale<any, any>) => this._extentsForScale(scale);
```

Comment:
I have to keep a reference to the wrapped function, because reference equality is used to add and remove `ExtentProvider`s (the entire point of this PR).